internal/ethapi/api.go

```diff
@@ -1471,7 +1472,8 @@ func (r *RollupTransaction) toTransaction(txNonce uint64) *types.Transaction {
 	if r.Target == nil {
 		tx = types.NewContractCreation(txNonce, big.NewInt(0), uint64(*r.GasLimit), big.NewInt(0), c, r.Sender, r.L1RollupTxId)
 	} else {
-		tx = types.NewTransaction(txNonce, *r.Target, big.NewInt(0), uint64(*r.GasLimit), big.NewInt(0), c, r.Sender, r.L1RollupTxId)
+		// TODO(mark): double check the sighash type here
```

Comment:
What should the sighash type here be? I'd think EIP155

Reply:
Yeah `EIP155` I believe